integration/src/test/java/org/mule/test/config/MuleConfigurationTestCase.java

```diff
@@ -36,6 +45,23 @@
   protected String workingDirectory = "target";
   private MuleContext muleContext;
 
+  private static final ClusterService mockedClusterService = mock(ClusterService.class);
```

Comment:
these two shouldn't be static. Have them as instance fields and use `@Mock` with the Mockito runner